src/routes-static.js

```diff
@@ -23,6 +23,10 @@ const getComponentLessonPage = (nextState, cb) => {
   cb(null, props => <Lesson {...props} lesson={result}/>);
 };
 
+const getComponent404Page = (nextState, cb) => {
+  cb(null, require('./pages/PageNotFound').PageNotFoundContainer);
+};
+
 
-const routes = getRouteObject(getComponentFrontPage, getComponentPlaylist, getComponentLessonPage);
+const routes = getRouteObject(getComponentFrontPage, getComponentPlaylist, getComponentLessonPage, getComponent404Page);
```

Comment:
Hvis jeg istedet for `yarn start`, kjører `yarn run build` og så `yarn run serve`, så fungerer ikke dette 404-opplegget. Jeg mistenker det er fordi routes-static.js ikke får med pathTest. Hadde det ikke vært bedre å bare definere denne funksjonen (og validPathTest) i routeObject.js?